SwiftCSV/Parser.swift

```diff
@@ -126,6 +126,12 @@ enum Parser {
     static func enumerateAsDict(header: [String], content: String, delimiter: CSVDelimiter, rowLimit: Int? = nil, block: @escaping ([String : String]) -> ()) throws {
 
         let enumeratedHeader = header.enumerated()
+        
+        // Check for duplicate column names
+        let duplicateColumns = header.duplicates()
+        if !duplicateColumns.isEmpty {
+            throw CSVParseError.generic(message: "Duplicate column names found: \(duplicateColumns.joined(separator: ", "))")
```

Comment:
Cool. I think this warrants a new error case 👍

`case duplicateColumns(columnNames: [String])` or something like it maybe.